pkg/platformconfig/platformconfig.go

```diff
@@ -104,6 +106,21 @@ func (config *Config) GetFunctionMetricSinks() (map[string]MetricSink, error) {
 	return config.getMetricSinks(config.Metrics.Functions)
 }
 
+func (config *Config) GetDefaultServiceType() v1.ServiceType {
+
+	// if not k8s, this field should be empty
```

Comment:
you can populate it for local platform as well, we can align behaviour for this specific purpose